### PR TITLE
The page was displaying as ```shell

### PR DESCRIPTION
**Pull request by dinukadev**
The page was displaying as ```shell  as we had an additional wrapper on top of it.

### CODE REVIEW

**Comment by dinukadev** [comment]
CLA signed

File: content/en/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -732,11 +732,10 @@ Eventually, resume the Deployment and observe a new ReplicaSet coming up with al
 kubectl rollout resume deployment.v1.apps/nginx-deployment
 ```
 deployment.apps/nginx-deployment resumed
```
Comment (Bradamant3) [inline content/en/docs/concepts/workloads/controllers/deployment.md]: You need to move this line into the code snippet at l732

File: content/en/docs/concepts/workloads/controllers/deployment.md [hunk 2]
```diff
@@ -732,11 +732,10 @@ Eventually, resume the Deployment and observe a new ReplicaSet coming up with al
...
-```
+
```
Comment (Bradamant3) [inline content/en/docs/concepts/workloads/controllers/deployment.md]: This cut is fine.

File: content/en/docs/concepts/workloads/controllers/deployment.md [hunk 3]
```diff
@@ -732,11 +732,10 @@ Eventually, resume the Deployment and observe a new ReplicaSet coming up with al
...
 ```shell
 kubectl get rs -w
 ```
-```
```
Comment (Bradamant3) [inline content/en/docs/concepts/workloads/controllers/deployment.md]: This cut breaks the snippet again. Take a look at the netlify preview.

**Comment by dinukadev** [comment]
@Bradamant3 thank you for the review. Appreciate it. I believe I have done fixes requested. Appreciate one more round of review to see if everything is good.

**Comment by cody-clark** [comment]
Hi @dinukadev, thanks for the PR! Can you take a look at the merge conflicts?

Ah, it looks like [13112](https://github.com/kubernetes/website/commit/0f45f093548693f7a5289292d0e9e446bbf7ae31#diff-d3460a8592a868b436ff807496e51d6f) addressed the same issue. 

I'm going to close this PR, but there's the same issue on [kubectl for Docker users](https://kubernetes.io/docs/reference/kubectl/docker-cli-to-kubectl/) if you're looking for another PR!

/close

**Comment by dinukadev** [comment]
Thanks @cody-clark

**Pull request closed without merging**